libqtile/window.py

```diff
@@ -556,6 +554,7 @@ def user_placed_window_setup(self, borderpixel, borderwidth):
         self.borderwidth = borderwidth
         self.bordercolor = borderpixel
         self.window.configure(borderwidth=borderwidth, stackmode=StackMode.Above)
+        # TODO: Could we handle this with a change of the z-layer? Is the window even part of a group?
```

Comment:
@m-col mentioned PR #1697 on IRC, which would remove this entire function anyway. I didn't bother inventing a fix because of that.